### ISSUE TITLE
Bouton `signalé` passe au dessus de la barre de défilement.

### ISSUE THREAD

**Issue by enimaloc** [readthedocs-fr/bin-server#136]
Lorsque un bin devient trop grand la barre de défilement apparaît sur la page, ce cas-ci n'est pas gérer pour le bouton `signalé` qui passe au-dessus de la barre de défilement.

![](https://user-images.githubusercontent.com/44118516/115126586-5d2a4f80-9fd0-11eb-89e9-2af2b27cf8fd.png)

**Comment by NoeMastrorillo** [comment]
À compter que c'est aussi le cas pour les boutons de controls
![image](https://user-images.githubusercontent.com/39499143/115255606-b7d9bd80-a12e-11eb-8c6b-979cc2cb9500.png)

**Comment by Mesteery** [comment]
Pour le boutons de contrôles, la scrollbar est présente inutilement des fois, c'est fix par #116

**Comment by NoeMastrorillo** [comment]
Le problème vient du fait que les boutons de contrôles ainsi que le bouton de signalement sont dans un conteneur différent de celui qui a les scroll-bar : 
- Pour `newform.html` la scrollbar appartient à la `textarea` tandis que les boutons de contrôles sont dans le `form`.
- Pour `highlight.html` les scrollbars appartiennent à `.highlight` tandis que les boutons de contrôles et le bouton de signalement sont dans le `body`.

Une solution serait de déplacer les boutons dans les conteneurs dont les scrollbars appartiennent, mais ça sera compliqué avec `.highlight` étant donné qu'il est généré par Pygments.

Une autre solution serait d'augmenter les padding/margin des boutons jusqu'à avoir une distance convenable de la scrollbar. Cependant les padding/margin resteront tels même si les scrollbars ne sont pas présentes, et je pense que la taille des scrollbars dépend du navigateur/appareil.

Est-ce que vous avez d'autres solutions ?

**Comment by Mesteery** [comment]
> Le problème vient du fait que les boutons de contrôles ainsi que le bouton de signalement sont dans un conteneur différent de celui qui a les scroll-bar :

Non, c'est juste qu'ils sont en position `fixed` / `absolute`.

**Comment by NoeMastrorillo** [comment]
> Non, c'est juste qu'ils sont en position `fixed` / `absolute`.

Oui, et bah s'ils étaient dans le même conteneur que celui qui a les scrollbars (avec `position: relative` sur le conteneur parent) on n'aurait pas ce problème là :
![image](https://user-images.githubusercontent.com/39499143/115287978-2d568580-a151-11eb-9887-243b8cf5af8c.png)

**Comment by NoeMastrorillo** [comment]
> Pour le boutons de contrôles, la scrollbar est présente inutilement des fois, c'est fix par #116

Quand je fetch ta PR c'est pas fix, il suffit juste d'enlever le `overflow-x: scroll` (d'ailleurs il fout quoi ici ???).

**Comment by Mesteery** [comment]
oui en fait je m'étais trompé cette PR ne le fix pas. Sinon, si, ils sert. (pour fix faut juste mettre `overflow-x: auto` au lieu de `scroll`)
